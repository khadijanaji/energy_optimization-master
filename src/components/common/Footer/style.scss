.footer{
  width: 100%;
  background-color: #485563;
  padding: 0 40px 0;
  font-family: Roboto;
  font-size: 10px;
  font-weight: normal;
  font-style: normal;
  font-stretch: normal;
  line-height: normal;
  letter-spacing: normal;
  z-index: 1;
  height: 54px;
  padding-top: 2px;
  &__txt1 {
    text-align: left;
    color: #ffffff;
    padding-top: 20px;

  }
  &__txt2 {
    text-align: center;
    color: #ffffff;
    padding-top: 20px;

  }
  &__txt3 {
    text-align: right;
    color: #99abbe;
    padding-top: 12px;
    &__img {
      height: 30px;
      
    }
  }
}