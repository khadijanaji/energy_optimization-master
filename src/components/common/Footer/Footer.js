// @flow

import * as React from "react";
import Grid from "@material-ui/core/Grid";
// Import STDigital from "../../../image/STDigital.png";

import "./style.scss";


const Footer = () => (

  <div className="footer">
    <Grid
      container
      spacing={2}
    >
      <Grid
        item
        xs
      >
        <div className="footer__txt1">&copy; 2019 All rights reserved</div>
      </Grid>
      <Grid
        item
        xs
      >
        <div className="footer__txt2">Powered by OCP</div>
      </Grid>
      <Grid
        item
        xs
      >
        <div className="footer__txt3">
          <img
            className="footer__txt3__img"
            // Src={STDigital}
          />
        </div>
      </Grid>

    </Grid>
  </div>
);

export default Footer;
