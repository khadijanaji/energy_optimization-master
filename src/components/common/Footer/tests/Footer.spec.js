/* globals describe it expect */

import React from "react";
import {shallow} from "enzyme";

import Footer from "../../Footer";

describe("<Footer />", () => {
  it("should display the Footer", () => {
    const wrapper = shallow(<Footer />);

    expect(wrapper).toMatchSnapshot();
  });
});
