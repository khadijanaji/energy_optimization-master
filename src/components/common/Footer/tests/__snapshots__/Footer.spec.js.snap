// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should display the Footer 1`] = `
<div
  className="footer"
>
  <WithStyles(Grid)
    container={true}
    spacing={2}
  >
    <WithStyles(Grid)
      item={true}
      xs={true}
    >
      <div
        className="footer__txt1"
      >
        Â© 2019 All rights reserved
      </div>
    </WithStyles(Grid)>
    <WithStyles(Grid)
      item={true}
      xs={true}
    >
      <div
        className="footer__txt2"
      >
        Powered by OCP
      </div>
    </WithStyles(Grid)>
    <WithStyles(Grid)
      item={true}
      xs={true}
    >
      <div
        className="footer__txt3"
      >
        <img
          className="footer__txt3__img"
        />
      </div>
    </WithStyles(Grid)>
  </WithStyles(Grid)>
</div>
`;
